test/data/split/single/150overlap4bins15window2errors.out

```diff
@@ -0,0 +1,10 @@
+0 start_position=890,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	
+1 start_position=7601,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	
+2 start_position=2395,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	
+3 start_position=1518,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	
+4 start_position=10047,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	
+5 start_position=4134,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	
+6 start_position=9638,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	
+7 start_position=16,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	
+8 start_position=9383,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	
+9 start_position=1278,length=150,errors=3,reference_id='1',reference_file='single_reference.fasta'	
```

Comment:
This file is created from an IBF that was built using input from the split segment metadata output file.